ALUM/ALUM/Components/Tags.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Tags.swift
```

Comment:
Let's change this filename to Tag.swift
Also, during your git stuff you might have missed the change which adds the file to project.pbxproj so make sure you do "Add file" using xcode.